File: bftotxt.py
def bfToPlainText(bfCode):
    output = ''
    memory = [0] * 30000
    pointer = 0

    for i in range(len(bfCode)):
        char = bfCode[i]

        if char == '>':
            pointer += 1
        elif char == '<':
            pointer -= 1
        elif char == '+':
            memory[pointer] += 1
        elif char == '-':
            memory[pointer] -= 1
        elif char == '.':
            output += chr(memory[pointer])
        elif char == ',':
            # Handle input if needed
            pass
        elif char == '[':
            if memory[pointer] == 0:
                loopCount = 1
                while loopCount > 0:
                    i += 1
                    if bfCode[i] == '[':
                        loopCount += 1
                    elif bfCode[i] == ']':
                        loopCount -= 1
        elif char == ']':
            if memory[pointer] != 0:
                loopCount = 1
                while loopCount > 0:
                    i -= 1
                    if bfCode[i] == ']':
                        loopCount += 1
                    elif bfCode[i] == '[':
                        loopCount -= 1

    return output

def runGetInput():
    bfCode = input('Enter BF code: ')
    output = bfToPlainText(bfCode)
    print(output)

runGetInput()

File: bftotxt.js
function bfToPlainText(bfCode) {
    let output = '';
    let memory = new Uint8Array(30000);
    let pointer = 0;

    for (let i = 0; i < bfCode.length; i++) {
        let char = bfCode[i];

        switch (char) {
            case '>':
                pointer++;
                break;
            case '<':
                pointer--;
                break;
            case '+':
                memory[pointer]++;
                break;
            case '-':
                memory[pointer]--;
                break;
            case '.':
                output += String.fromCharCode(memory[pointer]);
                break;
            case ',':
                // Handle input if needed
                break;
            case '[':
                if (memory[pointer] === 0) {
                    let loopCount = 1;
                    while (loopCount > 0) {
                        i++;
                        if (bfCode[i] === '[') loopCount++;
                        else if (bfCode[i] === ']') loopCount--;
                    }
                }
                break;
            case ']':
                if (memory[pointer] !== 0) {
                    let loopCount = 1;
                    while (loopCount > 0) {
                        i--;
                        if (bfCode[i] === ']') loopCount++;
                        else if (bfCode[i] === '[') loopCount--;
                    }
                }
                break;
        }
    }

    return output;
}

function runGetInput() {
    const bfCodeInput = document.getElementById("input").value;
    const StringedInput = document.getElementById("input2").value;
    document.getElementById("BFCode").innerHTML = `Result as output is equal to: ${bfToPlainText(bfCodeInput)}`;
}